index.html

```diff
@@ -138,14 +138,14 @@ <h1 class="f5 ttu tracked fw6">Principles</h1>
                 <article class="pv2 w-100 ">
                   <h2 class="f4 f1-l  fw6 mb0">Responsive</h2>
                   <p class="f5 f3-ns measure lh-copy mt0">
-                    Everything should be 100% responsive. Your website should work regardless of a users
-                    device or screensize.
+                    Everything should be 100% responsive. Your website should work regardless of a user's
```

Comment:
Thanks for the comment! Is `user` intended to be singular or plural? I read it as singular, hence not using the plural possessive (`users'`). Regardless, I just learned from @mrmrs that the docs template actually lives in `src/templates/index.html` so I'll go ahead and close this one out :)